Automations SDK Methods (#12830)
PrefectHQ/prefect: docs/concepts/automations.md

![Configuring an action for an automation in Prefect Cloud.](/img/ui/automations-action.png)

### Creating automations In Python code

> You can create and access any automation via the Python SDK via the `Automation` class and its methods.

Reviewer: ```suggestion
You can create and access any automation via the Python SDK, for example:
```

Reviewer: Double "via"
